app/javascript/controllers/letters_calls_form_controller.js

```diff
@@ -0,0 +1,94 @@
+// Get the value of an element attribute as a number or a default value
+function getAttributeAsNumber(element, attributeName, defaultValue = 0) {
+	return parseFloat(element.getAttribute(attributeName)) || defaultValue;
+}
+
+// Calculate the total based on the values of letters and phone_calls inputs
+function calculateTotal(lettersValue, phoneCallsValue, unitPriceForLetters, unitPriceForPhoneCalls) {
+	const total = lettersValue * unitPriceForLetters + phoneCallsValue * unitPriceForPhoneCalls;
```

Comment:
another very minor gripe - we don't use brackets here to separate out the two multiplications we are adding, but we use brackets on line 15 for an addition - not necessarily anything that needs changing but another case for us potentially bringing in linting as a way for us to have consistency

Reply:
changed already, simple math BODMAS. But Agree with you for the readability. changed accordingly.